### ISSUE TITLE
Koa Wrapper for this version?

### ISSUE THREAD

**Issue by MarcGodard** [node-oauth/node-oauth2-server#170]
Wondering if there is a Koa wrapper for this version. I think the one you link to is the old version and looks not to have been kept up with Koa releases too.

**Comment by jankapunkt** [comment]
Hi @MarcGodard at least I am not actively using Koa so I am not a good candidate to update the plugin. If you feel confident to help, then I can fork the linked one into this org and you can open a PR to update it.

@Uzlopak @HappyZombies @jorenvandeweyer one of you guys has experience with Koa for a potential review?

**Comment by MarcGodard** [comment]
I just started using Koa, so not confident enough yet, but I will look at the other wrapper (I did see a PR to update it for Koa v2 that was never merged but no pr for switching to this version). If I get the time, I can do it over the Christmas holidays and see what I can figure out.

I don't need this till January or February, just collecting and researching the tools I need to build the requested features.

I do like Koa more than express so far (started a couple weeks ago tho, but see myself continuing as express isn't really being updated in the right direction).
